juju/controller.py

```diff
@@ -249,6 +249,20 @@ def controller_uuid(self):
             await cloud_facade.UpdateCredentials(credentials=tagged_credentials)
         return name
 
+    async def add_cloud(self, name, cloud):
+        """Add a cloud to this controller.
+
+        :param str name: Name to give the new cloud.
+        :param Cloud cloud: Cloud configuration.
+        :return Cloud: Cloud that was created.
+        """
+        log.debug('Adding cloud %s', name)
+        print(self.connection().facades)
+        cloud_facade = client.CloudFacade.from_connection(self.connection())
+        print(cloud_facade.version)
```

Comment:
Remove print.

Reply:
oops, naughty debug prints think they can get into the commit